components/Layout.tsx

```diff
@@ -10,7 +10,7 @@ export default function Layout({ children }: { children: ReactNode }) {
     <div className="flex flex-col items-center justify-center min-h-screen bg-base-100 text-base-content">
       <Head>
         <title>{PUBLIC_SITE_TITLE}</title>
-        <link rel="icon" href="/juno.svg" />
+        <link rel="icon" href="/yin_yang.png" />
```

Comment:
Hope we can go back to svg for release

Reply:
https://github.com/DA0-DA0/cw-dao-dapp/issues/34